pages/Nix/Hyprland on Home Manager.md

```diff
@@ -62,3 +62,37 @@ in {
   };
 }
 ```
+### Pure evaluation
+```nix
+{ pkgs, ... }:
+let
+  flake-compat = builtins.fetchGit {
```

Comment:
Should be
```nix
  flake-compat = builtins.fetchTarball {
    url = "https://github.com/edolstra/flake-compat/archive/master.tar.gz"; # optionally replace with commit
    sha256 = ""; # see comment below
  };

  hyprland = (import flake-compat {
    src = builtins.fetchTarball {
      url = "https://github.com/hyprwm/Hyprland/archive/master.tar.gz"; # optionally replace "master" with a commit
      sha256 = ""; # not sure the builtin works like this, may need `lib.fakeHash256` instead of the empty string
    };
  }).defaultNix;
  ```